packages/block-editor/src/store/selectors.js

```diff
@@ -1837,10 +1836,14 @@ const getAllAllowedPatterns = createSelector(
 			// the raw blocks parser, also known as the "stage I" block parser.
 			// This is about 250x faster than the full parse that the Block API
 			// offers.
-			blockNodes: parseBlocks( pattern.content ),
+			__unstableBlockNodes: parse( pattern.content ),
```

Comment:
Should we use `__experimentalGetParsedPattern` selector then, that adds the `blocks` property? The comments needs to go away as well.

Reply:
Not sure I follow here :) would you mind pushing that change?

Reply:
Pushed: https://github.com/WordPress/gutenberg/pull/32376/commits/149e5e1def5cd32ac690c2904819446a88f2bdc3

Reply:
Thanks, very nice simplification.